api/links/links.py

```diff
@@ -127,11 +127,14 @@ class CreateLinkRequestModel(OPModel):
 
     input: str = Field(..., description="The ID of the input entity.")
     output: str = Field(..., description="The ID of the output entity.")
-    link: str = Field(
-        ...,
-        description="The name of the link type to create.",
+    name: str | None = Field(None, description="The name of the link.")
+    link: str | None = Field(
+        None,
+        description="Link type to create. This is deprecated. Use linkType instead.",
         example="reference|folder|version",
     )
+    link_type: str | None = Field(None, description="Link type to create.")
```

Comment:
MIght be worth having the `example` from `link` if they are the same.

In line 150 we'll fail if the `link_type` is not specified, so why allow `None` here?